src/FishjamClient.ts

```diff
@@ -662,8 +672,14 @@ export class FishjamClient<
     this.webrtc?.on('localTrackBandwidthSet', (event) => {
       this.emit('localTrackBandwidthSet', event);
     });
-    this.webrtc?.on('localTrackEncodingBandwidthSet', (event) => {
-      this.emit('localTrackEncodingBandwidthSet', event);
+    this.webrtc?.on('localTrackMuted', (event) => {
+      this.emit('localTrackMuted', event);
+    });
+    this.webrtc?.on('localTrackUnmuted', (event) => {
+      this.emit('localTrackUnmuted', event);
+    });
+    this.webrtc?.on('localTrackBandwidthSet', (event) => {
+      this.emit('localTrackBandwidthSet', event);
```

Comment:
This is outside of this PR scope, but I would like to discuss why FishjamClient class has so many functions 😮

Reply:
It is very convenient to use events to synchronize the state between TypeScript and the React client. Emitting events for every action allows us to react to every internal state change in the TypeScript client. This was introduced to simplify synchronization logic and to eliminate some race conditions